ruby/vampires.rb

```diff
@@ -0,0 +1,83 @@
+puts "how many employees will be processed?"
+employee_count = gets.chomp.to_i
+survey_count = 0
+
+while survey_count < employee_count
+survey_count += 1
+
+puts "What is your name?"
+user_name = gets.chomp
+
+
+
+puts "How old are you?"
+stated_age = gets.chomp.to_i
+
+puts "What year were you born?"
+birthYear = gets.chomp.to_i
+
+calculated_age = (2017 - birthYear)
```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.